libminifi/src/FlowController.cpp

```diff
@@ -163,6 +174,31 @@ FlowController::~FlowController() {
   provenance_repo_ = nullptr;
 }
 
+bool FlowController::applyConfiguration(std::string &configurePayload) {
```

Comment:
the FlowConfiguration does not have Flow Controller state, during reApply the config, somehow the FlowController need to expose a function so that listener can call to reapply  the config.